build/ci/windows/sign.bat

```diff
@@ -0,0 +1,66 @@
+@echo off
+ECHO "MuseScore sign"
+
+SET CERT_ENCRYPT_SECRET=""
+SET CERT_PASSWORD=""
+SET FILES_DIR=""
+SET FILE_PATH=""
+SET FILE_NAME=""
+
+SET SIGNTOOL="C:\Program Files (x86)\Windows Kits\10\bin\x64\signtool.exe"
+SET SIGNCERT="build\ci\windows\resources\musescore.p12"
+
+::SET TIMESERVER1="http://timestamp.globalsign.com/scripts/timstamp.dll"
```

Comment:
This one redirectes to an https page, https://www.globalsign.com/en/timestamp-service